stock_picking_report_valued_copy_unvalued/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+	:target: http://www.gnu.org/licenses/agpl
+	:alt: License: AGPL-3
+
+=======================================
+Stock Picking Report with Unvalued Copy
+=======================================
+
+This module allows to print an unvalued picking copy along with the valued picking in the delivery slip report
+
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Only install
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to Contact Form / Sales and Purchases. Configure the valued picking, and unvalued picking copy checks
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Print Delivery slip report.
+
+#. By default, you have the unvalued picking copy parameter configured on the partner. To edit it, you can go to the stock picking form / Additional info, and mark/unmark the "Unvalued picking copy" check
```

Comment:
Yo lo explicaría de otra manera. En plan, indicaría que el usuario puede realizar la configuración de la copia en el albarán individualmente pero que el parámetro viene dado por defecto de la ficha del contacto (no poner esto literalmente).

Reply:
¿Así mejor?:

Usage
=====

To use this module, you need to:

#. Configure the "unvalued picking copy" check on the partner form

#. Create a delivery stock picking

#. Print the delivery slip report

#. In the "Additional info" section of the picking, you can edit the "unvalued picking copy" option of without affecting its partner's default configuration option

Reply:
¿Así mejor?:

Usage
=====
To use this module, you need to:

#. Configure the "unvalued picking copy" check on the partner form

#. Create a delivery stock picking

#. Print the delivery slip report

#. In the "Additional info" section of the picking, you can edit the "unvalued picking copy" option without affecting its partner's default configuration option

Reply:
humm no se si indicar en el primer paso un (recordar que hay que tener activada la opción "valued picking").
#. Configure the "unvalued picking copy" check on the partner form -> indica si está en una page en que page está please.
Por el resto mejor :)

Reply:
Lo he cambiado, ¿así mejor?:

Usage
=====

To use this module, you need to:

#. Go to Contact Form / Sales and Purchases. Configure the valued picking, and unvalued picking copy checks

#. Create a delivery stock picking

#. Print the delivery slip report

#. In the "Additional info" section of the picking, you can edit the "unvalued picking copy" option without affecting its partner's default configuration option